test/emptytheme/styles/variation.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Nit: I wonder if we should try to make the variation more "meaningful". For instance add a "dark" variation to the empty theme.

Reply:
I can do it an a follow up to unblock the remaining functionality from waiting. I'd appreciate if some theme dev could   provide a minimal `dark` theme.json for empty theme. --cc @kjellr 🙏

Reply:
Hmm. So this would be just for the `emptytheme` version bundled here with Gutenberg, right? For the canonical version over in [theme-experiments](https://github.com/WordPress/theme-experiments/tree/master/emptytheme), I don't think adding a style makes sense since it's positioned primarily as an ultra-minimal starting point. 

It doesn't even specify a color palette, so making a dark mode version of it is a bit unexpected. 😄